docs/concepts/withdrawals.md

```diff
@@ -66,8 +66,10 @@ of Capella), there's no mechanism for recycling the validator ID that has been e
 From the first Capella slot, block proposers provide up to 16 withdrawals per
 proposed block.
 
-Proposers start from validator 1, and find validators that qualify: must have an
-Ethereum address as a withdrawal address, must have an excess balance, or have exited.
+Proposers start with validator 1 and search for validators that meet any of these criteria:
+- have an Ethereum address as a withdrawal address,
+- posses an excess balance,
+- have exited.
```

Comment:
```suggestion
Proposers start with validator 1, and find validators that have an Ethereum address as a withdrawal address, and meet one of these criteria:

- They have an excess balance.
- They have exited.
```

If you're unsure about the meaning, first read the context in the page to try to understand (that's what I did here to extract this list).

Reply:
Also, please don't use synonyms when consistent, simple words suffice (i.e., "have" and "possess").

Reply:
@joaniefromtheblock can you take a look at these comments?